classes/fields/boolean.php

```diff
@@ -174,10 +174,7 @@ public function data( $name, $value = null, $options = null, $pod = null, $id =
 	 * {@inheritdoc}
 	 */
 	public function validate( $value, $name = null, $options = null, $fields = null, $pod = null, $id = null, $params = null ) {
-
-		$required = (int) pods_v( 'required', $options, 0 );
-
-		if ( ! $required ) {
+		if ( ! $this->is_required( $options ); ) {
```

Comment:
* :no_entry_sign: PHP syntax error: syntax error, unexpected ';'
* :no_entry_sign: Inline control structures are not allowed

<sub>hash: cc78a7d30c181fb0af408d9fa4877076</sub>